package/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.16.0
+FROM registry.suse.com/bci/bci-base:15.3
```

Comment:
Can we get the full image tag. The current one is `15.3.17.17.13`.

There are two reasons I'm asking for this...
1. We'll know the exact version of the BCI being used at any given time. This gives us visibility (e.g., if we ever need to troubleshoot)
2. There won't be a different version between dev and production. Let's say we do our work and testing to one version. Then, before release there is a new version of 15.3. The release would be built on that instead of the tested version. While I would not expect issues, this provides a little more control.

Reply:
Like I said before, that version is not a valid semver and I am afraid that dependabot can't update it. Do you know if that works in other projects?

Reply:
Dependabot does work with the long version. I tested it - https://github.com/rancher/shell/pull/28.

Reply:
I'm not sure what problems that could be, given that we could replace the OS with a `scratch` image.